src/lib/registration/connect_helpers.rb

```diff
@@ -215,6 +215,20 @@ def self.ask_import_ssl_certificate(cert)
     end
 
     def self.import_ssl_certificate(cert)
+      # Has been a certificate already imported? In some cases the certificate
+      # import might not help, avoid endless certificate import loop.
+      if Storage::InstallationOptions.instance.imported_cert_sha256_fingerprint
+        # TRANSLATORS: multiline error message - a SSL certificate has been
+        # imported but the registration server still cannot be accessed securely,
+        # user has to solve the certificate issue manually.
+        Yast::Report.Error(_("A certificate has been already imported\n" \
+          "but the server connection still cannot be trusted.\n\n" \
+          "Please fix the certificate issue manually, ensure that the server\n" \
+          "can be connected securely and start the YaST module again."))
+
+        return false
```

Comment:
It seems useful to document the return value, because we've made a subtle change.
Before:
> \# @return [Boolean] certificate successfully imported

Now:
> \# @return [Boolean] success? can fail if cannot import or the cert is not valid after all